modules/encryption.md

```diff
@@ -0,0 +1,71 @@
+# Encryption 
+
+The `encrypted` key is a Link Property.
+
+The `encrypted` key contains an Encryption Object that indicates how a given resource has been encrypted or obfuscated, and provides relevant information for decryption.
+
+## Encryption Object
+
+The Encryption Object has the following keys:
+
+| Key   | Semantics | Type     | Required? |
+| ----- | --------- | -------- | --------- |
+| [algorithm](#algorithm)  | Identifier of the algorithm used to encrypt the resource.  | URI  | Yes |
+| [scheme](#scheme)  | Identifier of the encryption scheme used to encrypt the resource.  | URI  | No |
+| [profile](#profile)  | Identifier of the encryption profile used to encrypt the resource.  | URI  | No |
+| [compression](#compression)  | Compression method used on the resource.  | String | No |
+| [originalLength](#originalLength)  | Original length of the resource in bytes before compression and/or encryption. | Integer  | No |
+
+
+## LCP Encrypted Resource
+
+Publication of any type can be protected by the [Readium LCP](https://readium.org/lcp-specs/releases/lcp/latest) DRM.
+
+In such a case, allowed values for the properties of the Encryption Object are:
+
+| Property    | Value | 
+| ----------- | ----- | 
+| scheme      | `http://readium.org/2014/01/lcp` |
+| profile     | see the [LCP Encryption Profile Registry](https://readium.org/lcp-specs/registries/profiles) |
+| algorithm   | `http://www.w3.org/2001/04/xmlenc#aes256-cbc` |
+| compression | `deflate` |
```

Comment:
It reads like LCP requires deflated resources, which is not the case.

```suggestion
| compression | `deflate` or `null` |
```

Reply:
I have changed the wording to make clear that compression is used in certain use cases only.